src/System.Text.JsonLab/System/Text/Json/JsonWriter.cs

```diff
@@ -0,0 +1,1539 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Almost none of this code is new. Most of it is just merging the JsonWriterUtf8 and JsonWriterUtf16 files.